server/preload.js

```diff
@@ -84,6 +84,11 @@ contextBridge.exposeInMainWorld('API', {
     logWarning: (msg, optionalData) => ipcRenderer.send('log-warn', msg, optionalData),
     logError: (msg, optionalData) => ipcRenderer.send('log-error', msg, optionalData),
 
+    // Updater
+    checkForUpdates: () => ipcRenderer.invoke('check-for-updates'),
+    onUpdateAvailable: (callback) => ipcRenderer.on('update-available', callback),
+    updateAction: (updateAction) => ipcRenderer.send('update-action', updateAction),
```

Comment:
I added these two, but they are not implemented yet. They would allow to implement a UI instead having the dialogs in case you want. Basically I will call `onUpdateAvailable` when there is an update available. and I'll send the version of the update and the change logs. And then you can call `updateAction` with the action, we can decide what that action is, but it should mean: `Download and Install`, `Ignore until the next check`, `Ignore until app restart`, `Ignore this version`.